docker/entrypoint.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+echo << EOL
+-------------------------------------
+          ____           ___
+         / __/__ _______/ (_)
+        / _// -_) __/ _  / /
+      _/_/  \__/_/  \_,_/_/
+     / __/__ _____  _____ ____
+    _\ \/ -_) __/ |/ / -_) __/
+   /___/\__/_/  |___/\__/_/
+Brought to you by getferdi.com
+Support our Open Collective at:
+https://opencollective.com/getferdi/
+EOL
+
+# if config.txt doesn't exist then create one with default values
+if [ ! -f /config/config.txt ]; then	
+	cp /app/.env.example /config/config.txt
+fi
+
+# use config.txt default values as .env file	
+if [ -f /app/.env ]; then	
+	rm /app/.env	
+	ln -s /config/config.txt /app/.env	
+elif [ ! -f /app/.env ]; then	
+	ln -s /config/config.txt /app/.env	
+fi
```

Comment:
I'd just change this to:
```suggestion
if [ -f /config/config.txt ]; then
    cp /config/config.txt /app/.env
else
    cp /app/.env.example /app/.env
fi
```

Because we are read-only anyway.

Reply:
Fixed in [https://github.com/getferdi/server/commit/0a3ce5a9e37dd917cfabdf97fdc18a41d88c65eb](https://github.com/getferdi/server/commit/0a3ce5a9e37dd917cfabdf97fdc18a41d88c65eb)